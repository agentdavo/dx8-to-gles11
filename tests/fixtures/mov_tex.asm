mov oD0, v0
mov oT1, v1
