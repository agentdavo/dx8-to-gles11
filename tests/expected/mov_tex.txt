COLOR4F
MULTITEXCOORD4F stage=1
